lahja/common.py

```diff
@@ -36,11 +36,13 @@ def __init__(
         filter_endpoint: Optional[str] = None,
         filter_event_id: Optional[RequestID] = None,
         internal: bool = False,
+        raise_if_no_subscribers: bool = True,
     ) -> None:
 
         self.filter_endpoint = filter_endpoint
         self.filter_event_id = filter_event_id
         self.internal = internal
+        self.raise_if_no_subscribers = raise_if_no_subscribers
```

Comment:
@pipermerriam I'm almost certain you'll have an idea for a better name for this :sweat_smile:

Reply:
A good policy is to look for the name that doesn't have any contradictions.  The `no_subscribers` is the part that I think sparks the most confusion here.

maybe:

- `require_active_subscription`
- `require_subscriber`
- `is_subscription_required`
- `is_subscriber_required`

Any of those do it for you?

Reply:
Ha, `require_subscriber` is so obvious now that you bring it up :see_no_evil: